src/__tests__/__snapshots__/CloseButton.js.snap

```diff
@@ -12,6 +12,7 @@ exports[`CloseButton renders without props 1`] = `
   display: inline-block;
   vertical-align: middle;
   text-align: center;
+  -webkit-text-decoration: none;
```

Comment:
Stylis added a bunch of vendor prefixes, so be sure to `npm install` to get the latest for snapshot tests

https://github.com/thysultan/stylis.js/issues/95